### PR TITLE
Tweaks for Living CR and REC

### PR DESCRIPTION
**Pull request by plehegar**
This matches the proposed charters for Service Workers and Browser Testing and Tools.

1. Success criteria: don't assume the document will move to Proposed Recommendation
2. Success criteria: should expect intent of implementations for changes if no Proposed Recommendation
3. Patent Policy: s/Recommendations/Web specifications/

### CODE REVIEW

**Comment by plehegar** [comment]
fyi @dontcallmedom

**Review by svgeesus — APPROVED** [review]
Yes, happy with all these changes

File: charter-template.html [hunk 1]
```diff
@@ -167,7 +167,8 @@ <h2>
 
         <p>More detailed milestones and updated publication schedules are available on the <a href="https://www.w3.org/wiki/[groupname]/PubStatus">group publication status page</a>.</p>
 
-        <p><i>Draft state</i> indicates the state of the deliverable at the time of the charter approval. <i>Expected completion</i> indicates when the deliverable is projected to become a Recommendation, or otherwise reach a stable state.</p>
+        <p><i>Draft state</i> indicates the state of the deliverable at the time of the charter approval. <span class='todo'>Choose one:</span> <i>Expected completion</i> indicates when the deliverable is projected to become a Recommendation, or otherwise reach a stable state
+          <span class='todo'>The Working Group intends to publish the latest state of their work as Candidate Recommendation and does not intend to advance their documents to Recommendation (no expected milestones)</span>.</p>
```
Comment (wseltzer) [inline charter-template.html:171]: s/Candidate Recommendation/Candidate Recommendation (with Snapshots)/

**Review by wseltzer — APPROVED** [review]
with a suggested change

File: charter-template.html (hunk 1 above)
Comment (wseltzer) [reply charter-template.html:171]: pr #345

**Pull request merged**